sway-ir/src/printer.rs

```diff
@@ -549,6 +549,21 @@ fn instruction_to_doc<'a>(
                 ))
                 .append(md_namer.md_idx_to_doc(context, metadata)),
             ),
+            Instruction::Log {
+                log_val,
+                log_ty,
+                log_id,
+            } => maybe_constant_to_doc(context, md_namer, namer, log_val)
+                .append(maybe_constant_to_doc(context, md_namer, namer, log_id))
+                .append(Doc::line(
+                    Doc::text(format!(
+                        "log {} {}, {}",
```

Comment:
No comma after the type?  Feels inconsistent -- but then I took a quick look elsewhere in the file and there is no consistency already. 🙁 We did consider removing commas but they're needed for multi-word args like types.  This still needs to be looked at one day.

Reply:
I was honestly just following `ret`. `ret` and `log` are pretty much the same except for the extra ID parameter and the fact that `log` does not terminate the program.